docs/guides/image-optimization/imageurl-image-optimization.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Use Image Optimization with imageUrl
+description: Learn how to migrate an application using Auth.js to use Clerk for authentication.
+---
+
+# Use Image Optimization with `imageUrl`
+
```

Comment:
It would be nice to explain what we mean about image optimization and why they should use it. Ie:

Clerk allows users to upload their own custom profile image, which app developers can fetch and display. Clerk strongly recommends that developers who use this feature also use optimization parameters that specify the minimum size and quality necessary for their use case. Smaller values ensure smaller images are pulled thereby reducing the total amount of network bandwidth required to load your app and improving overall page load performance.

Reply:
Done 👍